app/src/main/res/values/strings.xml

```diff
@@ -44,4 +44,13 @@
     <string name="feedback">Feedback</string>
     <string name="this_is_cover">This is cover facebook</string>
     <string name="this_is_avatar">This is avatar facebook user</string>
+    <string name="double_back">Please click BACK again to exit!</string>
+    <string name="no_results_found">No results found!</string>
+    <string name="no_favorite_movies">You haven\'t favored any movies!</string>
+    <string name="truyen_mail">tdtruyen92lmq@gmail.com</string>
+    <string name="thang_mail">huynhthanhthang1296@gmail.com</string>
+    <string name="subject_email">Report bugs</string>
+    <string name="content_email">I want to report bugs\n\n- Movie: \n\n- My device: \n\n- I have a problem: </string>
+    <string name="title_choice_app">Send email...</string>
```

Comment:
WARNING: [TypographyEllipsis] Replace "..." with ellipsis character (…, &#8230;) ?
 You can replace the string "..." with a dedicated ellipsis character, ellipsis character (…, &#8230;). This can help make the text more readable.